script.js

```diff
@@ -328,9 +382,23 @@ const usesDiagram = {
   type: "bar",
   value: "#age#",
   xAxis: {
-    title: "Age",
-    template: "#age#"
+    title: "Country",
+    template: function(obj) {
+      if (obj.$group) return obj.country;
+      else return obj.age;
+    }
   },
+  ready: function() {
+    $$(this).sync($$("usersList"), function() {
```

Comment:
dont use `sync` in the `ready` handler, it is not called if the widget does not have the initial data. 
in your code the handler is called by chance, because you have set the url property, which is incorrect, as chart should npot have any own data

Reply:
wrapping sync code into one more `webix.ready` is incorrect, as it does not give any benefit. you'd better handle `body.unload` (which is webix.ready) once and include all the logic inside: 

~~~js
webix.ready(function(){
    webix.ui(...);
    view.sync(collection);
});
~~~